### PR TITLE
Support `latest` version

### PR DESCRIPTION
**Pull request by k1LoW**
Hi @k2tzumi 
~~How about setting the default version to the latest version as the runn-action?~~ Allow `latest` version to be specified.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ inputs:
   version:
     description: Specify runn version
     required: false
-    default: "v0.114.1"
+    default: latest
```
Comment (k2tzumi) [inline action.yml:21]: > How about setting the default version to the latest version as the runn-action?

The version of runn-action is linked to the version of runn
This version is updated when tagging by running the following job
If runn-action's latest version is specified, runn's latest version is expected to be used if the version is published in conjunction with runn-action.
https://github.com/k2tzumi/runn-action/blob/main/.github/workflows/release.yml

Comment (k1LoW) [reply action.yml:21]: Understood 👍 

On the other hand, it seems a waste not to be able to specify `latest`.

Comment (k1LoW) [reply action.yml:21]: Also, it looks like you should update the `v0` and `latest` tags.

Comment (k2tzumi) [reply action.yml:21]: I see
I guess I'll add a process to re-cut the v0 and latest tags in the workflow at the time of release.

Comment (k1LoW) [reply action.yml:21]: Thank you!

Comment (k1LoW) [reply action.yml]: This PR only allows to specify the `latest` version.

Comment (k2tzumi) [reply action.yml]: Issue registered
https://github.com/k2tzumi/runn-action/issues/42
https://github.com/k2tzumi/runn-action/issues/43

**Pull request merged**